src/line-chart.js

```diff
@@ -288,6 +288,7 @@ dc.lineChart = function (parent, chartGroup) {
                     .attr('r', getDotRadius())
                     .style('fill-opacity', _dataPointFillOpacity)
                     .style('stroke-opacity', _dataPointStrokeOpacity)
+                    .attr('fill', _chart.getColor)
```

Comment:
FYI, without this the dots will be black and transition to the color. I caught this visually but [this spec](https://github.com/dc-js/dc.js/blob/master/spec/line-chart-spec.js#L462) will also catch it.